dotnet-engine/Yggdrasil.Engine/Types.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Yggdrasil;
 
-public class Context
+public sealed class Context
```

Comment:
I've marked a bunch of publicly exposed classes as sealed. This is solving 0 problems outside of crazy people deriving these classes but I'd rather stop them now than have to stop them later. If they never do, then this does nothing